var/spack/repos/builtin/packages/root/package.py

```diff
@@ -21,23 +21,25 @@ class Root(CMakePackage):
         branch='master')
 
     # Development versions
-    version('6.13.08', '2b4546fc4b712ba1d672401bee07f5b3')
+    version('6.15.02', sha256='2236fe4935139459239c935b2f93b3aa6bbfe92765ae9d9db9dd0b947bf19071')
```

Comment:
are these two mixed up?

Reply:
No, the old development version was moved to the list of old releases along with other past development releases. I agree that the plural "Development versions" comment is misleading though, will gladly change it if you like.

Reply:
Is there a reason v6.13.08 is being removed?

Reply:
It is not being removed, only moved to the "old releases" section as it is not the current development release anymore :smile: